modules/core/shared/src/main/scala/lucuma/core/model/ids.scala

```diff
@@ -6,9 +6,9 @@ package lucuma.core.model
 import lucuma.core.util.WithGid
 import lucuma.refined.*
 
-object Configuration  extends WithGid('c'.refined)
```

Comment:
No `Configuration`?

Reply:
I couldn't find any reference to it and believe it is a relic from the past when we had top-level objects (identified with a GID) for configurations and constraints.

Reply:
I'll leave it open until tomorrow or so in case anybody wants to cry foul.